src/helpers/update-pull-requests.ts

```diff
@@ -27,20 +28,30 @@ function isIssueEvent(event: object): event is IssueEvent {
 
 export async function updatePullRequests(context: Context) {
   const { logger } = context;
+  const results: ResultInfo[] = [];
+
   if (!context.config.repos.monitor.length) {
-    return logger.info("No organizations or repo have been specified, skipping.");
+    const owner = context.payload.repository.owner;
+    if (owner) {
+      logger.info(`No organizations or repo have been specified, will default to the organization owner: ${owner.login}.`);
+    } else {
+      return logger.error("Could not set a default organization to watch, skipping.");
+    }
   }
 
   const pullRequests = await getOpenPullRequests(context, context.config.repos);
 
   if (!pullRequests?.length) {
     return logger.info("Nothing to do.");
   }
+
+  let isMerged = false;
```

Comment:
Since this is only used in the loop and reset on each iteration is there any reason not to define it just inside the loop?